specifyweb/backend/workbench/tests.py

```
@@ -237,4 +239,102 @@ def test_preserve_imported_data_and_mapping(self) -> None:
         self.assertEqual(dataset["uploadplan"], uploadplan)
-        
\ No newline at end of file
+
+class ChangeOwnershipTests(ApiTests):
+    def setUp(self) -> None:
+        super().setUp()
+        self.new_owner = Specifyuser.objects.create(
+            isloggedin=False,
+            isloggedinreport=False,
+            name="newowner",
+            password="unused",
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "password"

(S106)